views/layout.dt

```diff
@@ -16,7 +16,7 @@ html
 			.helper
 				.helper.expand-container.active
 					.logo
-						a(href=".")
+						a(href="..")
```

Comment:
It wouldn't matter for code.dlang.org, but since this might be run in an intranet setting and possibly put under a subfolder using a reverse proxy setting, rather `#{req.rootDir}` should be used.